inc/saiport.h

```diff
@@ -572,6 +587,15 @@ typedef enum _sai_port_attr_t
     SAI_PORT_ATTR_INTERNAL_LOOPBACK_MODE,
 
     /**
+     * @brief Forward Error Correction (FEC) control
+     *
+     * @type sai_port_fec_mode_t
+     * @flags CREATE_AND_SET
+     * @default SAI_PORT_FEC_MODE_NONE
```

Comment:
I have the same understanding. When AN is enabled, FEC mode should be resolved based on the capability advertised by the PHY as defined in standard. This attribute should define the mode for cases wherever AN is disabled.